README.md

```diff
@@ -4,5 +4,5 @@ Android Dagger Demo
 Testing with Dagger 2, Espresso 2 and Mockito.
 
 Blog posts: 
-   * http://blog.sqisland.com/2015/04/dagger-2-espresso-2-mockito.html
```

Comment:
Can you keep this link, and add the mock application link as the 2nd bullet item? The first link is still relevant to understand the Mockito part. Thanks!